recipes/qrupdate/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+make lib FC=${FC} FFLAGS="${FFLAGS} -fimplicit-none -O3 -funroll-loops" PREFIX=${PREFIX}
+make solib FC=${FC} FFLAGS="${FFLAGS} -fimplicit-none -O3 -funroll-loops" LDFLAGS="${LDFLAGS}" PREFIX=${PREFIX}
+
+if [ `uname` == Darwin ]; then
+    make -B install FC=${FC} FFLAGS="${FFLAGS} -fimplicit-none -O3 -funroll-loops" LDFLAGS="${LDFLAGS}" PREFIX=${PREFIX}
+else
+    make install PREFIX=${PREFIX}
+fi
```

Comment:
Is there a `make check` we can run here?

Reply:
No, but we have a `make test`:
```
help:
	@echo
	@echo "The following targets are available:"
	@echo "   make help    - displays this help"
	@echo "   make lib     - compiles a static library"
	@echo "   make solib   - compiles a dynamic library"
	@echo "   make test    - compiles and runs the testsuite"
	@echo "   make clean   - cleans up everything"
	@echo "   make install - installs everything"
```

Reply:
Let's run it!

Reply:
Is that the proper way in 79134aa?

Reply:
Or can just add after `solib` in the previous line?

```
make solib FC=${FC} FFLAGS="${FFLAGS} -fimplicit-none -O3 -funroll-loops" LDFLAGS="${LDFLAGS}" PREFIX=${PREFIX}
 make test 
```

Reply:
No idea. We need to check the makefile.  I believe it must be called before make install though since it seems to compiled everything for the tests.

Reply:
> No idea. We need to check the makefile. I believe it must be called before make install though since it seems to compiled everything for the tests.

So from testing locally, you basically have to call it separately. Though we need to add yet another flag as per Unidata/netcdf-fortran#212

Reply:
Ugh... the test passes okay locally but it doesn't here!

Reply:
Alright, this is not worth my time. Closing this PR.